### PR TITLE
feat(pattern/aspectratios): add documentation

### PR DESCRIPTION
**Pull request by caqueste**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/adeo/design-system--styleguide/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

### CODE REVIEW

**Comment by melanieanglardLMFR** [comment]
@caqueste, i think you should add part about responsive behavior. Maybe we should add a code part given an example about how to implement responsive ratio using picture tag and so on ... 

@gael-boyenval what do you think ?

File: src/pages/Foundations/AspectRatios/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: 'Aspect ratios'
+order: 10
+---
+
+> The aspect ratio of an image is **the proportional relationship between its width and its height**. Aspect ratios are commonly written as a formula of width to height, like “16:9” or “4:3". The first value corresponds to the width, and the second to the height. <br>
+> For example, an image with an aspect ratio of 1:1 is a square. So, the width and the height are identical, like 80px x 80px.
+
+## Utility
+
+For designers, by using aspect ratio for images, and applying it on yours products, you will be able to create consistency between all the pictures.<br>
```
Comment (gael-boyenval) [inline src/pages/Foundations/AspectRatios/index.md]: I don't thinks this all section is useful. I would just remove it, or add something very small like : 

"aspect ration bring consistent image sizes across product and viewports"

File: src/pages/Foundations/AspectRatios/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: 'Aspect ratios'
```
Comment (gael-boyenval) [inline src/pages/Foundations/AspectRatios/index.md]: Images aspect ratios

File: src/pages/Foundations/AspectRatios/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Utility
+
+For designers, by using aspect ratio for images, and applying it on yours products, you will be able to create consistency between all the pictures.<br>
+For other roles, reference image formats and use them in our user interface designs allows :
+
+- Better anticipation of the image display. It is easier to predict which area of the image will be visible when cropping.
+- Time saving for developers. It is no longer necessary for them to use specific values for each image.
+
+## Recommended ratios
```
Comment (gael-boyenval) [inline src/pages/Foundations/AspectRatios/index.md]: authorized or recommended ?

File: src/pages/Foundations/AspectRatios/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+Here are the **6 recommended ratios** for building your components :
+
+- 16:9
+- 3:2
+- 4:3
+- 1:1
+- 3:4
+- 2:3
```
Comment (gael-boyenval) [inline src/pages/Foundations/AspectRatios/index.md]: ok so pretty much all aspect ratios ?

File: src/pages/Foundations/AspectRatios/index.md (hunk 3 above)
Comment (nherwegh) [reply src/pages/Foundations/AspectRatios/index.md]: it is recommended

**Pull request merged**